File: javascript/bee1045.js
const input = require("fs").readFileSync("stdin", "utf-8")
const lines = input.split(" ")

let A = Number(lines.shift())
let B = Number(lines.shift())
let C = Number(lines.shift())

if(A < B || A < C) {
    if( B <= C) {
        var variavel = A
        A = C
        C = variavel
    } else {
        var variavel = A
        A = B
        B = variavel
    }
}

if(A >= (B + C)) {
    console.log("NAO FORMA TRIANGULO")
} else if((A * A) == ((B * B) + (C * C))){
    console.log("TRIANGULO RETANGULO")
} else if((A * A) > ((B * B) + (C * C))) {
    console.log("TRIANGULO OBTUSANGULO")
} else if((A * A) < ((B * B) + (C * C))) {
    console.log("TRIANGULO ACUTANGULO")
}

if(A == B && A == C) {
    console.log("TRIANGULO EQUILATERO")
} else if(A == B || A == C || B == C) {
    console.log("TRIANGULO ISOSCELES")
}

File: python/bee1045.py
a, b, c = input().split()
a = float(a)
b = float(b)
c = float(c)

if a < b or a < c:
    if b <= c:
        variavel = a
        a = c
        c = variavel
    else:
        variavel = a
        a = b
        b = variavel

if a >= (b+c):
    print("NAO FORMA TRIANGULO")
elif (a * a) == ((b * b) + (c * c)):
    print("TRIANGULO RETANGULO")
elif (a * a) > ((b * b) + (c * c)):
    print("TRIANGULO OBTUSANGULO")
elif (a * a) < ((b * b) + (c * c)):
    print("TRIANGULO ACUTANGULO")

if a == b and a == c:
    print("TRIANGULO EQUILATERO")
elif a == b or a == c or b == c:
    print("TRIANGULO ISOSCELES")
